.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
VSC is using its own copy of TS which bite me for a moment - we always want it to use the installed one. Even though it's an editor config file (which doesnt normally belong to git) I find it highly useful to include it so it doesnt trip anyone in the future.